React/Base/RCTUtils.m

```diff
@@ -733,8 +733,13 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
   return [directory stringByAppendingPathComponent:filename];
 }
 
-static void RCTGetRGBAColorComponents(CGColorRef color, CGFloat rgba[4])
+void RCTGetRGBAColorComponents(CGColorRef color, CGFloat rgba[4])
```

Comment:
Is this an expensive operation? Should we try to normalize the UIColor to this and then use it across the whole border rendering operation?